### ISSUE TITLE
Discussion 3rd party user/channel/permission management

### ISSUE THREAD

**Issue by bgadrian** [inspircd/inspircd#361]
Hello,

Before looking more into inspircd I need to know if the following feature can be done 

 it supports (module or other way) to be integrated to an 3rd party application, on short 

: I want to block any login/register from the normal usage way, I want with another application to create/change password for users accounts, and give them permissions to specific channels (and ofc to manage the channels).

Basically to use it as chat system in an application, with more layers of rooms and privigiles.

Thanks,
Adrian
